man/aptly.1

```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "APTLY" "1" "November 2016" "" ""
+.TH "APTLY" "1" "2017-01-10" "" ""
```

Comment:
I didn't finish putting `CONTRIBUTING` notes yet, but upstream `ronn` has one bug (https://github.com/rtomayko/ronn/pull/83) which is not patched. It leads to broken rendering of man page (and it's the reason for huge diff here).

I've got patched version here: https://github.com/smira/ronn

Probably I need to put some Makefile magic to pull in patched ronn and do transformation.